db/migrate/20180821190417_create_accounts.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAccounts < ActiveRecord::Migration
+  def change
+    create_table :accounts do |t|
+      t.uuid :uuid, null: false, unique: true
+      t.string :idme_uuid, unique: true
+      t.string :icn
```

Comment:
Should this be unique too?

Reply:
Good question.  We did consider that.  Our reasoning for not adding validation for `icn` uniqueness, at either the db layer or the model layer, is to keep things flexible for now.

At this current juncture, there is no use case for the `icn` nor the `edipi`.  Just the `uuid`. 
 We are adding them forecasting that they may/should be useful in the future.  So until we have locked down precisely how we will use the `icn`, we want to keep this flexible.